src/reactionHandlers/remove.test.ts

```diff
@@ -0,0 +1,12 @@
+import { _add, allReactionHandlers } from './remove';
+
+describe('allReactionHandlers', () => {
+	test('index is not empty', () => {
+		expect(allReactionHandlers.size).toBeGreaterThan(0);
+	});
+
+	test('commands can be added', () => {
+		// eslint-disable-next-line no-empty-function, @typescript-eslint/no-empty-function
+		expect(() => _add({ execute: () => {} } as unknown as ReactionHandler)).not.toThrow(TypeError);
```

Comment:
ESLint isn't happy with arrow functions that end in `{}`, in part because it's unclear whether the intent is a function that returns `void` (or `undefined`) or a function that returns an empty object. Returning `undefined` explicitly makes this clear to our tooling:

```suggestion
		expect(() => _add({ execute: () => undefined })).not.toThrow(TypeError);
```